Makefile

```
@@ -0,0 +1,27 @@
+.PHONY: build clean test lint
+
+build: magedbm2.phar
+
+clean:
+	rm -rf vendor box.phar composer.phar magedbm2.phar
+
+test: install
+	php -derror_reporting=E_ERROR ./vendor/bin/phpunit
+
+install: composer.phar composer.lock
+	./composer.phar install
+
+lint: install
+	./vendor/bin/phpcs --standard=./phpcs.xml src/ || exit 0
+	./vendor/bin/phpmd src/ text ./phpmd.xml || exit 0
+	./vendor/bin/phpcbf --standard=./phpcs.xml src/ || exit 0
+
+magedbm2.phar: * box.phar composer.phar
```

Do we need `test` or `lint` as dependencies of building the package?

```suggestion
magedbm2.phar: install box.phar composer.phar
```

---

That `*` was there to ensure that the target was rebuilt if anything in the directory changed -- I think that's what we want? Or maybe just `src/*`?

---

You're right. I'm just being a numpty!